config/resources/domain.json

```diff
@@ -255,6 +256,54 @@
       },
       "features": ["include-uri"],
       "new-resource-base": "http://lblod.data.gift/snippet-lists/"
+    },
+    "published-snippet-containers": {
+      "name": "published-snippet-container",
+      "class": "ext:PublishedSnippetContainer",
+      "relationships": {
+        "current-version": {
+          "predicate": "pav:hasCurrentVersion",
+          "target": "published-snippet",
+          "cardinality": "one"
+        },
+        "versions": {
+          "predicate": "pav:hasVersion",
+          "target": "published-snippet",
+          "cardinality": "many"
+        },
+        "derived-from": {
+          "predicate": "prov:derivedFrom",
+          "target": "document-container",
+          "cardinality": "one"
+        }
+      },
+      "new-resource-base": "http://lblod.data.gift/published-snippet-containers/"
+    },
+    "published-snippets": {
+      "name": "published-snippet",
+      "class": "ext:PublishedSnippet",
+      "relationships": {
```

Comment:
this doesn't have a link to the previous version
I think its quite important to have that